TestResultSummaryService/parsers/Test.js

```diff
@@ -46,17 +55,34 @@ class Test extends Parser {
             }
             if (testName) {
                 testStr += line + "\n";
+                // decide whether to run the Perf test
+                goToBenchmarkParser = BenchmarkParser.canParse(benchmarkName, benchmarkVariant);

```

Comment:
Don't think we need the `canParse` in `BenchmarkParser`. We should just check the pipeline name to see whether it has `perf` in it as mentioned in https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/136#issuecomment-545496000. If it does, then we need to additionally call the `BenchmarkParser`.

Reply:
Though checking build name wouldn't work when we use grinder (doesn't have "perf" in its name) or we run multiple tests that include both functional and perf tests, something that doesn't happen often since they are usually run separately on different machines. @llxia @smlambert Any recommendations?